include/CppUTest/CommandLineTestRunner.h

```diff
@@ -32,6 +32,7 @@
 #include "TestOutput.h"
 #include "CommandLineArguments.h"
 #include "TestFilter.h"
+#include <cwchar>
```

Comment:
Direct dependencies with Lib C aren't allowed in CppUTest. You'd need to wrap the methods in PlatformSpecific to deal with portability to "less supported" platforms.